extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -550,28 +550,31 @@ private static function getMediaDetailWithSizeFallback(
 		$fallbackSize = self::MAX_MERCURY_CONTENT_WIDTH
 	) {
 		$mediaDetail = WikiaFileHelper::getMediaDetail( $title, $mediaDetailConfig );
-		if ( empty( $mediaDetail['width'] ) ) {
-			$mediaDetail['width'] = $fallbackSize;
 
-			\Wikia\Logger\WikiaLogger::instance()->error(
-				'ArticleAsJson - Media width was empty - fallback to fallbackSize',
-				[
-					'media_details' => $mediaDetail,
-					'fallback_size' => $fallbackSize
-				]
-			);
-		}
+		if ( !empty( $mediaDetail['mediaType'] ) ) {
```

Comment:
I suggest using `$mediaDetail['exists'] === true`, that's what it's for.
https://github.com/Wikia/app/blob/dev/includes/wikia/services/WikiaFileHelper.class.php#L311